### PR TITLE
manage the draft status from Gihub PR

### PR DESCRIPTION
**Pull request by flepied**

### CODE REVIEW

File: monocle/github/pullrequest.py [hunk 1]
```diff
@@ -296,6 +297,7 @@ def extract_pr_objects(pr):
             change = {}
             change['type'] = 'Change'
             change['id'] = pr['id']
+            change['draft'] = pr['isDraft']
```
Comment (morucci) [inline monocle/github/pullrequest.py:300]: You need to add that new field in the schema.

Comment (morucci) [reply monocle/github/pullrequest.py:300]: Could you add an issue to add the same support in Gerrit ?

Comment (flepied) [reply monocle/github/pullrequest.py:300]: is the same feature available in gerrit?

Comment (morucci) [reply monocle/github/pullrequest.py:300]: Seems there was a draft support that has been removed. There is a "work in progress"  https://www.gerritcodereview.com/2.15.html#support-for-draft-changes-removed
Well let's manage that if when someone will require it.

Comment (morucci) [reply monocle/github/pullrequest.py:300]: In https://github.com/morucci/monocle/blob/master/monocle/db/db.py#L101 add "draft": {"type": "boolean"},

**Pull request merged**